### PR TITLE
DEV: Depreciate Support For Python 3.7.

### PR DESCRIPTION
**Pull request by shaneahmed**
- Remove support for python 3.7 to allow using latest version of image codecs and some advanced features.

### CODE REVIEW

**Comment by shaneahmed** [comment]
@John-P Please can you also check this https://github.com/TissueImageAnalytics/tiatoolbox/blob/d68b446d658349c210e5b0a2bfc98458e26dc177/tiatoolbox/annotation/storage.py#L155 in addition to reviewing this PR?

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ language: python
 python:
   - 3.9
   - 3.8
```
Comment (John-P) [inline .travis.yml:6]: We could possibly add in 3.10 here now

**Review by John-P — APPROVED** [review]
Looks good. We could consider adding 3.10 support now also.

**Comment by shaneahmed** [comment]
Python 3.10 not yet supported by openslide. https://github.com/openslide/openslide-python/issues/141

**Comment by shaneahmed** [comment]
> Looks good. We could consider adding 3.10 support now also.

Python 3.10 not yet supported by openslide. https://github.com/openslide/openslide-python/issues/141

**Pull request merged**